pkg/cmd/ctlstore/main.go

```diff
@@ -58,7 +58,7 @@ type reflectorCliConfig struct {
 	PollJitterCoefficient float64            `conf:"poll-jitter-coefficient" help:"Coefficient for poll jittering"`
 	QueryBlockSize        int                `conf:"query-block-size" help:"Number of ledger entries to get at once"`
 	Debug                 bool               `conf:"debug" help:"Turns on debug logging"`
-	LedgerHealth          ledgerHealthConfig `conf:"ledger-latency" help:"Configure ledger latency behavior"`
```

Comment:
There are a number of things using `ledger-latency` in their configurations. While I think that this change is good, it will probably break some things unless we change it elsewhere in lockstep.

Reply:
make sense. I was just a bit confused by the term ledger-latency while we were passing a ledgerHeath config :)